### DISCUSSION TITLE
[XWindow] Only show title of window on active display

### DISCUSSION THREAD

**Discussion by 0xf0xx0** [polybar/polybar#2734] [Ideas]
I'm using a multi-monitor setup and ive noticed the `xwindow` module on all of them displays the title of the active window, ignoring the display the window is on. It would be nice for it to only show the title when the display the bar is on is active, and keep the title when the user moves to a different monitor. I've attached a mockup below.

![xwindow_mockup](https://user-images.githubusercontent.com/33764485/170889154-61cc3ee5-c171-473e-97d2-ad391854acb8.png)

**Comment by patrick96** [comment]
This has already been requested in #252

**Reply by 0xf0xx0** [reply]
Oh whoops, must've missed it when searching.
